.travis.yml

```diff
@@ -32,6 +32,10 @@ matrix:
       env:
         - SYMFONY_VERSION='2.7.*@dev'
         - FRAMEWORK_EXTRA_VERSION='~3.0'
+    - php: 5.6
+      env:
+        - PHPUNIT_FLAGS="--coverage-clover=coverage.clover"
+        - COVERAGE=true
 
 # only needed for the hackaround for https://github.com/symfony/symfony/issues/12868
```

Comment:
@dbu just require `symfony/phpunit-bridge` and your ready. Then, fix all deprecations. After this, you're probably left with a few deprecations because:

1. Legacy classes are used to remain BC
2. Things can't be fixed nicely (the usage of `factory-service` in `services.xml` for instance)

For (1), you have to annotate your tests/testcases with `@group legacy` and for (2), you use the env variable `SYMFONY_DEPRECATION_HELPER=weak`.